istioctl/pkg/writer/ztunnel/configdump/connections.go

```diff
@@ -66,13 +66,12 @@ func (c *ConfigWriter) PrintConnectionsSummary(filter ConnectionsFilter) error {
 	d := c.ztunnelDump
 	serviceNames := map[string]string{}
 	workloadNames := map[string]string{}
-	for netIP, s := range d.Services {
-		_, ip, _ := strings.Cut(netIP, "/")
+	for _, s := range d.Services {
+		_, ip, _ := strings.Cut(s.Addresses[0], "/")
```

Comment:
Services may not all have a VIP (which was actually a motivation for the change) so this could panic. Can we make this fallback to "none" or something?

Reply:
so we should have some tests on this

Reply:
I think this can be covered in e2e tests in a follow up PR

Reply:
Tests, yes. but can we make it not panic now?

Reply:
Duh, panic can happen when `s.Addresses` is nil and we're trying to index into it?

Will update!